panel/param.py

```diff
@@ -573,6 +579,15 @@ def action(event):
     def _ordered_params(self):
         params = [(p, pobj) for p, pobj in self.object.param.objects('existing').items()
                   if p in self.parameters or p == 'name']
+        if self.sorted is True or callable(self.sorted):
```

Comment:
```python
if self.sort:
```

Should be sufficient.

Reply:
It depends. If we can change to `param.ClassSelector` to get validation then yes. Otherwise if user provided `sorted="no"` I implemented it to ignore it and use the default sorting. With your suggestion it would instead sort alphabetically. :-)

Reply:
See above :-)

Reply:
You already did it, ignore me :-)